src/Modules/Family/EventSlotsModalComponent.js

```diff
@@ -55,8 +55,8 @@ const EventSlotsModalComponent = (props) => {
         <Modal.Body>
           <div className="container">
             {eventHour.map((item, index) => (
-              <div className="" key={'ul' + index}>
-                {item.event_slots.map((e, i) => (
+              item.event_slots.filter((e, i) => (e.capacity > 0)).map((e, i) => {
```

Comment:
need to filter event_slots if open_slots  are available

so, can u replace `e.capacity > 0` with `open_slots > 0`

Reply:
Oh Yeah done